src/player/client/modals/Settings/index.jsx

```diff
@@ -235,7 +287,13 @@ export default observer(function Settings(props) {
                 </div>
               </div>
               <div className="contact-us">
-                <Button ghost onClick={handleContactUs}>
+                <Button
+                  ghost
+                  onClick={
+                    handleContactUs &
```

Comment:
wtf ? is it working ?
```suggestion
                   ()=> { handleContactUs();
```